src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * rentPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * rentPrice) - smallDiscount;
+  }
+
+  if (days >= 7) {
+    return (days * rentPrice) - bigDiscount;
+  }
```

Comment:
You don't need if statement here.
Both previous options cover every possible different path, so feel free to return without `if` after them